### PR TITLE
added find a bird sellection for looping.

### PR DESCRIPTION
**Pull request by maigctical**

### CODE REVIEW

**Comment by pmallory** [comment]
Please review this, @casey-collab, the holiday weekend is over!

File: english/find-bird/bird.md [hunk 1]
```diff
@@ -15,3 +15,5 @@ Are lifted [to freedom](../marshmallow.md) Or so you hope...
 Enjoy the [ride](../enjoy-DreamRide/ride.md)
 
 Pinch [yourself.](../is-this-a-dream/is-this-a-dream.md)
+
+look for [find a bird.](find-bird/bird.md)
```
Comment (casey-collab) [inline english/find-bird/bird.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by pmallory** [comment]
This PR is getting stale, feel free to open a new PR if you're still interested in making a contribution to the story. Thanks!

**Pull request closed without merging**